frontend/utilities/software_install_scripts.ts

```diff
@@ -29,6 +29,7 @@ const getDefaultInstallScript = (fileName: string): string => {
     case "tar.gz":
     case "sh":
     case "ps1":
+    case "ipa":
```

Comment:
Fixes a miss on my part causing client side error to appear when trying to upload an IPA